README.md

```diff
@@ -1,24 +1,82 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
 
-Things you may want to cover:
+## Recipe App
+This application keeps track of an user's transactions for different categories. It Allows a user to save categories, create transactions or delete categories, and Keeps track of the total transactions under a given category.
 
-* Ruby version
+## See live version
 
-* System dependencies
+[Live version](https://still-coast-22556.herokuapp.com/)
 
-* Configuration
+## Presentation by Author
 
-* Database creation
+[Video Presentation](https://www.loom.com/share/aa0f946aaeff462e8c0bddbac4696ca5) 
 
-* Database initialization
+## Built With
 
-* How to run the test suite
+- Technologies used:
+  - Ruby
+  - Rails
+  - css
+  - postgresql
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+## Getting Started
 
-* ...
+### Prerequisites
+
+You need to have a browser and this tools installed:
+
+- Git
+- Ruby
+- MVC patterns
+
+### Setup
+
+To get a local copy up and running follow these simple example steps:
+
+```
+$ git clone https://github.com/Shubh-Dev/budget-navigant2.git
+```
+
+```
+$ cd budget-navigant2
+```
+
+```
+```
+
+```
+$ bundle install
+```
+
+### Usage
+
+To run the server:
+
+```
+$ rails s
+```
+
+## Author
+
+👤 <a href="https://github.com/Shubh-Dev" target="blank"><img align="center"
+      src="https://avatars.githubusercontent.com/u/46110284?v=4"
+      alt="Shubh-Dev" height="50" width="50"/></a> **Shubh-Dev**
+
+- GitHub: [@Shubh-dev](https://github.com/Shubh-Dev)
+- LinkedIn: [Shubh Dev](https://www.linkedin.com/in/shubhscb/)
+
+## Acknowledgement
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] Kindly give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).